pkg/labels/matcher.go

```diff
@@ -86,3 +87,16 @@ func (m *Matcher) Matches(s string) bool {
 	}
 	panic("labels.Matcher.Matches: invalid match type")
 }
+
+// openMetricsEscape is similar to the usual string escaping, but more
+// restricted. It merely replaces a new-line character with '\n', a double-quote
+// character with '\"', and a backslack with '\\', which is the escaping used by
```

Comment:
Typo : backslash

Reply:
You see how damaged I am by Slack already… :-/